api/maven-api-core/src/main/java/org/apache/maven/api/feature/Features.java

```diff
@@ -62,6 +64,18 @@ public static boolean buildConsumer(@Nullable Session session) {
         return buildConsumer(session != null ? session.getUserProperties() : null);
     }
 
+    public static boolean xinclude(@Nullable Properties userProperties) {
+        return doGet(userProperties, XINCLUDE, true);
```

Comment:
I would still advocate for this being off by default (opt-in) instead of on by default (opt-out) since XML parsers have   a long history of security vulnerabilities surrounding this stuff.

Reply:
Done